plugins/net_plugin/net_plugin.cpp

```diff
@@ -1745,7 +1760,10 @@ namespace eosio {
          c->close( false, true );
          return;
       }
-      c->consecutive_rejected_blocks = 0;
+      c->strand.post( [c, blk_num]() {
```

Comment:
This `strand.post`  no longer needed.

Reply:
Made `consecutive_rejected_blocks` atomic to allow removal of this `strand.post`.